chat-commands.js

```diff
@@ -613,15 +613,15 @@ exports.commands = {
 			targetRoom.chatRoomData.isPrivate = true;
 			Rooms.global.writeChatRoomData();
 			if (Rooms.get('upperstaff')) {
-				Rooms.get('upperstaff').add(`|raw|<div class="broadcast-green">Private chat room created: <b>${Chat.escapeHTML(target)}</b></div>`).update();
+				Monitor.adminlog(`|raw|<div class="broadcast-green">Private chat room created: <b>${Chat.escapeHTML(target)}</b></div>`);
```

Comment:
The issue with this is the monitors add a message to the rooms from the user `~` not as raw HTML. Even if its edited to be `/html` instead of `|raw` i think the current implementation __looks__ better in chat. Also adminlog logs to console too.

Reply:
Ah